### ISSUE TITLE
Unable to test AOM (accessibility object model)

### ISSUE THREAD

**Issue by gsnedders** [web-platform-tests/wpt#8355]
This needs some way to touch the accessibility layer, which in a sense is related to #7753.

**Comment by foolip** [comment]
Not yet known to involve infra changes, so changing labels.

**Comment by zcorpan** [comment]
The WebDriver spec is getting some new primitives here: https://github.com/w3c/webdriver/pull/1444

We could [extend testdriver.js](https://web-platform-tests.org/writing-tests/testdriver-extension-tutorial.html) to use the new WebDriver commands, which would enable testing computed role and computed label.

cc @alice @jugglinmike @spectranaut @AutomatedTester

**Comment by cookiecrook** [comment]
Can probably close this now that #38758 has landed and #38925 is in review. Other issues here: https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues
